types/codemirror/addon/display/autorefresh.d.ts

```diff
@@ -0,0 +1,16 @@
+// Type definitions for CodeMirror
+// Project: https://github.com/marijnh/CodeMirror
+// Definitions by: mtgto <https://github.com/mtgto>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+// See docs https://codemirror.net/doc/manual.html#addon_autorefresh
+
+import * as CodeMirror from "codemirror";
+
+declare module "codemirror" {
+    interface EditorConfiguration {
+        // if true, it will be refreshed the first time the editor becomes visible.
+        // you can pass delay (msec) time as polling duration
+        autoRefresh?: boolean | { delay: number };
```

Comment:
Can we make this property nullable? 🤔 
[This source](https://codemirror.net/addon/display/autorefresh.js) assigns `null` to the `cm.state.autoRefresh` like following: 

```javascript
    if (cm.state.autoRefresh) {
      stopListening(cm, cm.state.autoRefresh)
      cm.state.autoRefresh = null
    } 
```

Do you know whether is this assignment allowed to addon users?

Reply:
[Document](https://codemirror.net/doc/manual.html#addon_autorefresh) says autoRefresh is optional param either `true` or `{ delay: xxx (msec) }` object.
Implementation of autorefresh addons, you notice, is nullable.
IMO, We should not be nullable because doc does not mention it can be null.

Reply:
@azoson My opinion is non-nullable.
But I don't have confident.
Can you choose nullable or non-nullable?

Reply:
hmm... @typescript-bot requests to me "new comment or commit", but nothing changes after I commented.